src/util/GitHub.ts

```diff
@@ -16,6 +16,7 @@ export interface CommunityPlugin {
 
 export type Release = {
 	url: string;
+	manifest_url: string;
```

Comment:
I would like to keep this as the original GitHub Api Response https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-a-release

Reply:
Alright. I'll get to working on that as soon as I can. That was done to keep my changes to your code as minimal as possible, but I have another idea for implementing the patch

Reply:
Change implemented. Running the linter and builder, then testing the resulting plugin right now.

Reply:
Linter and build checks have passed: https://github.com/anonhostpi/vare-obsidian-plugin/pull/3

Testing the plugin now.